platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/Transform.java

```diff
@@ -151,6 +151,11 @@ CameraPosition invalidateCameraPosition() {
         cameraChangeDispatcher.onCameraMove();
       }
 
+      if (this.cameraPosition != null && (this.cameraPosition.tilt != cameraPosition.tilt
```

Comment:
Expression hard to read. What about breaking it down introducing explaining variables?